torch/csrc/jit/passes/onnx/constant_map.cpp

```diff
@@ -205,6 +205,22 @@ c10::optional<c10::SymbolicShape> ConstantValueMap::GetShapeValue(
   return ConstantValueMap::getInstance().shapeValueMap[tensorName];
 }
 
+void ConstantValueMap::SetGeneratedShape(std::unordered_map<std::string, ::ONNX_NAMESPACE::TensorShapeProto> generated_shape) {
+  ConstantValueMap::getInstance().generatedShapeDataByName = generated_shape;
```

Comment:
Makes sense. Then, I decide to use generatedShapeDataByName  for member variable name to make it consistent with ONNX so the function here will be `GetGeneratedShapeDataByName `. I eventually decide to remove setter function because giving call by reference initially, there is no need to call a setter function for generatedShapeDataByName and SymbolDimMap.